pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -1115,3 +1283,61 @@ func prepareStoreWithDeltaSnapshots(storeContainer string, numDeltaSnapshots int
 
 	return store
 }
+
+// addObjectsToStore adds objects to the given store. It creates and saves objects based on the provided parameters.
+// The objectType can be either "Chunk" or "Composite". The kind specifies the type of the snapshot Full/delta
+// The startRevision and lastRevision define the revision range of the objects.
+// The numOfSnapshots determines the number of objects to be created.
+// The creationTime specifies the creation time of the object
+func addObjectsToStore(store brtypes.SnapStore, objectType string, kind string, startRevision int, lastRevision int, numOfSnapshots int, creationTime time.Time) {
+	var isChunk bool
+	if objectType == "Chunk" {
+		isChunk = true
+	}
+	for i := 0; i < numOfSnapshots; i++ {
+		snap := brtypes.Snapshot{
+			Kind:          kind,
+			CreatedOn:     creationTime,
+			StartRevision: int64(startRevision),
+			LastRevision:  int64(lastRevision),
+			IsChunk:       isChunk,
+		}
+		if !snap.IsChunk {
+			snap.GenerateSnapshotName()
+		} else {
+			partNumber := i + 1
+			snap.SnapName = fmt.Sprintf("%s-%08d-%08d-%d/%010d%s%s", snap.Kind, snap.StartRevision, snap.LastRevision, snap.CreatedOn.Unix(), partNumber, snap.CompressionSuffix, "")
+		}
+		err := store.Save(snap, io.NopCloser(strings.NewReader(fmt.Sprintf("dummy-snapshot-content for snap created on %s", snap.CreatedOn))))
+		Expect(err).NotTo(HaveOccurred())
+	}
+}
+
+// getStoreAndSnapshotterFromConfigs returns a new local snapstore and new snapshotter
+func getStoreAndSnapshotterFromConfigs(container string, config *brtypes.SnapshotterConfig, etcdConnectionConfig *brtypes.EtcdConnectionConfig, compressionConfig *compressor.CompressionConfig, healthConfig *brtypes.HealthConfig, storeConfig *brtypes.SnapstoreConfig) (brtypes.SnapStore, *Snapshotter) {
+	snapstoreConf := &brtypes.SnapstoreConfig{Container: path.Join(outputDir, container), Prefix: "v2"}
+	store, err := snapstore.GetSnapstore(snapstoreConf)
+	Expect(err).NotTo(HaveOccurred())
+
+	ssr, err := NewSnapshotter(logger, config, store, etcdConnectionConfig, compressionConfig, healthConfig, storeConfig)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Although these are helping functions but IMO they shouldn't have statement like this `Expect(err).NotTo(HaveOccurred())`, this should be done by caller not by function itself